app/Http/Controllers/CategoryController.php

```diff
@@ -1,111 +0,0 @@
-<?php
-
-namespace App\Http\Controllers;
-
-use Illuminate\Http\Request;
-use App\Category;
-
-class CategoryController extends Controller
-{
-
-    public function __construct()
-    {
-        $this->middleware('auth');
-    }
-
-    /**
-     * Display a listing of the resource.
-     *
-     * @return \Illuminate\Http\Response
-     */
-    public function index()
-    {
-        $this->authorize('viewAny', Category::class);
-        // display a view of all of our categories
```

Comment:
Only place comments where necessary. Reading the code should make things clear most of the time.